docs/contributing.rst

```diff
@@ -11,6 +11,19 @@ Contributing to Scrapy
 
 There are many ways to contribute to Scrapy. Here are some of them:
 
+* Contribute to Scrapy by fixing currently available issues in the repository.
+  Scrapy has a list of `good first issues`_ and `help wanted issues`_ that you
+  can work on. These issues are a great way to get started with contributing to
+  Scrapy. If you're new to the codebase, you may want to focus on documentation
+  or testing-related issues, as they are always useful and can help you get
+  more familiar with the project. You can also check Scrapy's `test coverage`_
+  to see which areas may benefit from more tests.
```

Comment:
I think this point is technically covered below in this list (submit patches). It may be best to merge both bullet points into a single one, and most of the details should go into the actual section later on the page, to keep the list short and to the point.

The idea is that the list shows which different ways you can contribute, and later sections expand on *how* you can contribute in any of those ways.

Reply:
I changed the format of the doc a little, creating a new header for a section called "Contributing to Scrapy codebase", as the previous one mentioned many other non-code-related ways, and making it not a header would make the next sections smaller. Not sure about the formatting, but something along these lines I think could be a good way to introduce the codebase and everything encompassed by it.

Reply:
One thing that would be good for newcomers is to have a section teaching how to set up properly a development environment, _e.g_. Create a virtualenv, installing tests requirements, etc.

Reply:
For this one I believe we should create a new issue, as it probably will require a brand new rst for it. Nevertheless, it certainly should be done.